server/api_server/mcp_endpoint.py

```
@@ -1,379 +1,975 @@
+        except json.JSONDecodeError:
+            content.append({
+                "type": "text",
+                "text": api_response.text
+            })
+
+        is_error = api_response.status_code >= 400
+
+        return {
+            "content": content,
+            "isError": is_error
+        }
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)